examples/fewshot_protonet/README.md

```diff
@@ -0,0 +1,101 @@
+# Prototypical Networks for Few-shot Learning
+
+### 1. Description
+
+This example demonstrates how to implement [Prototypical Networks for Few-shot Learning](https://github.com/jakesnell/prototypical-networks) within the PyKale pipeline using PyTorch-Lightning. Below, we offer guidance on how to use ProtoNet effectively.
+
+ProtoNet is designed for the few-shot learning under N-Way-K-Shot setting.
```

Comment:
N and K are numbers so they should have dollar signs like $N$ and $K$.

Reply:
English
Either: for the few-shot learning scenarios under the $N$-Way-$K$-Shot setting
or: for few-shot learning under the $N$-Way-$K$-Shot setting

Reply:
Changed

Reply:
Have you checked your changes https://github.com/pykale/pykale/tree/add_prototypical_network/examples/fewshot_protonet? 
You need to not only make changes but also check before requesting for review.

Reply:
Remove the dash in between.

Reply:
Removed